src/ophys_etl/modules/segmentation/graph_utils/feature_vector_segmentation.py

```diff
@@ -0,0 +1,630 @@
+import matplotlib.pyplot as plt
+
+from typing import Optional, List, Tuple
+import networkx as nx
+import numpy as np
+import multiprocessing
+import multiprocessing.managers
+import h5py
+import pathlib
+import time
+import json
+
+from ophys_etl.types import ExtractROI
+from ophys_etl.modules.decrosstalk.ophys_plane import OphysROI
+
+from ophys_etl.modules.segmentation.graph_utils.feature_vector_rois import (
+    PearsonFeatureROI)
+
+import logging
+
+logger = logging.getLogger(__name__)
+logging.captureWarnings(True)
+logging.basicConfig(level=logging.INFO)
+
+
+def graph_to_img(graph_path: pathlib.Path,
+                 attribute: str = 'filtered_hnc_Gaussian') -> np.ndarray:
+    """
+    Convert a graph into a np.ndarray image
+
+    Parameters
+    ----------
+    graph_path: pathlib.Path
+        Path to graph pickle file
+
+    attribute: str
+        Name of the attribute used to create the image
+        (default = 'filtered_hnc_Gaussian')
+
+    Returns
+    -------
+    np.ndarray
+        An image in which the value of each pixel is the
+        sum of the edge weights connected to that node in
+        the graph.
+    """
+    graph = nx.read_gpickle(graph_path)
+    node_coords = np.array(graph.nodes).T
+    row_max = node_coords[0].max()
+    col_max = node_coords[1].max()
+    img = np.zeros((row_max+1, col_max+1), dtype=float)
+    for node in graph.nodes:
+        vals = [graph[node][i][attribute] for i in graph.neighbors(node)]
```

Comment:
so far, I've only seen edges having attirbutes, not nodes. perhaps I have not gotten there yet.

Reply:
oh, I see, nevermind, I guess:
`graph[i[[j]["name"]` is another way to see the (i, j) edge attribute